lib/config.js

```diff
@@ -1434,9 +1449,11 @@ util.stripYamlComments = function(fileStr) {
  * @protected
  * @method stripComments
  * @param fileString {string} The string to strip comments from
+ * @param stringRegex {RegExp} Optional regular expression to match strings
```

Comment:
Could you provide a better explanation?  Is this regex supposed to match comments, non-comments, full lines, partial lines?